source/features/conversation-activity-filter.tsx

```diff
@@ -41,7 +41,7 @@ function processSimpleComment(item: HTMLElement): void {
 	}
 
 	// Hide comments marked as resolved/hidden
-	if (select.exists('.minimized-comment:not(.d-none) > details', item)) {
+	if (select.exists('.minimized-comment > details', item)) {
```

Comment:
Also fixes the feature for regular comments that have been minimized.

Reply:
I think the class excludes non-minimized comments. 

In short, GitHub (used to?) has 2 "comment" elements for each comment. One is the regular version and one is the minimized version: They use that class to hide one of them, so effectively we use it to find the comments that are actually minimized (as opposed to DOM elements that aren't shown)

Reply:
Apparently that's no longer the case, because only comments that are actually minimized will have that class now. In fact the `> details` part is not even really needed anymore.